src/components/coin.ts

```diff
@@ -135,6 +140,37 @@ export const changeDbCoinBalanceByUserId = async (
   await createCoinLedgerEntry(userId, oldBalance, newBalance, event, reason, adminId);
 };
 
+/*
+  Get the leaderboard for the current coin amounts.
+*/
+export const getCurrentCoinLeaderboard = async (limit = 10): Promise<UserCoinEntry[]> => {
+  const db = await openDB();
+  const res = await db.all(
+    `
+      SELECT user_id, balance
+      FROM user_coin
+      ORDER BY balance DESC
+      LIMIT ${limit}
```

Comment:
In the past I used "?"
instead of ${}
is there a big difference? Just curious

Reply:
Yup, it is recommended to use "?" to avoid SQL injections.

Reply:
Nope, you guys are right, I'll change it to use "?"